alfresco-transform-tika/alfresco-transform-tika-boot/src/test/resources/quick.odf_metadata.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Now that the raw metadata "subject" is multi line (two in these cases) there is a  "Pangram, fox, dog, " to the previously expected value.